packages/frontend/src/components/common/Filters/FilterInputs/DefaultFilterInputs.tsx

```diff
@@ -112,6 +186,7 @@ const DefaultFilterInputs = <T extends ConditionalRule>({
                     className={disabled ? 'disabled-filter' : ''}
                     disabled={disabled}
                     fill
+                    placeholder={placeholder}
```

Comment:
@joaoviana  are you applying this placeholders only on DefaultFilterInputs or is there going to be another PR for for example DateFilterInputs?

Reply:
Thanks for raising this @IrakliJani, I'll be doing this on this PR!